documentation/samplefiles/readme.go.md

```diff
@@ -12,35 +12,15 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
-  - tag: package-2019-12-01
-  - tag: package-2020-07-01-preview
-  # add every tag listed below
+  - tag: package--[[Version]]-[[ReleaseState]]
 ```
 
-### Tag: package-2019-12-01 and go
+### Tag: package-[[Version]]-[[ReleaseState]] and go
 
-These settings apply only when `--tag=package-2019-12-01 --go` is specified on the command line.
-Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+These settings apply only when `--tag=package-[[Version]] --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
```

Comment:
This folder root directory variable name in go SDK code generator is slightly different with other languages.
```suggestion
Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
```